spec/project/object/root_object_spec.rb

```diff
@@ -43,6 +43,10 @@ module ProjectSpecs
       @root_object.minimized_project_reference_proxies.should == '0'
     end
 
+    it 'returns the preferred project object version' do
+      @root_object.preferred_project_object_version.should == '73'
```

Comment:
I suspect we will want to check that this is in a set like the `LAST_KNOWN_OBJECT_VERSION`